modules/phase_field/src/ics/IsolatedBoundingBoxIC.C

```diff
@@ -0,0 +1,133 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "IsolatedBoundingBoxIC.h"
+#include "MooseMesh.h"
+#include <iostream>
+
+registerMooseObject("PhaseFieldApp", IsolatedBoundingBoxIC);
+
+namespace sizeVector_def_isolated
```

Comment:
This should just be an anonymous namespace since the enclosed function is only used in this current file (compilation unit).